spec/support/shared_examples/behaves_like_valid_activity_xml.rb

```diff
@@ -20,9 +20,13 @@
 
   it "contains the funding organisation XML" do
     visit organisation_activity_path(organisation, activity, format: :xml)
-    expect(xml.at("iati-activity/participating-org[@role = '1']/@ref").text).to eq(activity.funding_organisation_reference)
-    expect(xml.at("iati-activity/participating-org[@role = '1']/@type").text).to eq(activity.funding_organisation_type)
-    expect(xml.at("iati-activity/participating-org[@role = '1']/narrative").text).to eq(activity.funding_organisation_name)
+    if activity.funding_organisation.present?
```

Comment:
Not sure how else you'd do this with shared examples, but we now have a test "contains the funding organisation XML" which in the case of a fund, expects that it doesn't.